x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -42,7 +43,7 @@ export const getNetworkTopNFlowColumns = (
       const ipAttr = `${topNFlowType}.ip`;
       const ip: string | null = get(ipAttr, node);
       const id = escapeDataProviderId(
-        `networkTopNFlow-table--${topNFlowType}-${topNFlowDirection}-ip-${ip}`
+        `${tableId}-table--${topNFlowType}-${topNFlowDirection}-ip-${ip}`
```

Comment:
Nothing wrong with the double `dash` but it looks like in other places you prefer single dash for the ids. Pointing it out in case you want all single dashes to make this look Swweeeettttt 🍰